assets/CMakeLists.txt

```diff
@@ -4,14 +4,18 @@ if(UNIX AND NOT APPLE)
     include(KDEInstallDirs)
 
     foreach(_i 16 24 32 64 128)
-        install(FILES "freedesktop/appicons/${_i}/zeal.png"
+        install(FILES "freedesktop/appicons/${_i}/org.zealdocs.Zeal.png"
```

Comment:
AFAIU this will break all icon themes, no? Plus Zeal itself tries to load icon `zeal` from the current theme.

Reply:
It shouldn't, but I can rename it on flatpak side as well (especially if it's also in the code). I will revert this chunk after work.

Reply:
If it's some a standard/recommendation, I think we can switch, but I haven't seen other apps using such icon names.